package.json

```diff
@@ -125,11 +126,12 @@
     "docs": "webpack --config .config/webpack.config.js",
     "eslint": "clear && eslint ./ && echo \"No errors found.\"",
     "eslint:watch": "esw -w --clear --cache",
+    "precommit": "yarn dist && ./tasks/pre-commit/index.sh",
```

Comment:
`./tasks/pre-commit/index.sh` should be replaced with:
```
git diff --exit-code dist docs
```
and you can remove `./tasks/pre-commit/index.sh` file